addon/components/materialize-collection-links.js

```diff
@@ -0,0 +1,6 @@
+import materializeCollection from './materialize-collection';
+
+export default materializeCollection.extend({
+  layoutName: 'components/materialize-collection',
+  tagName: 'div',
```

Comment:
you never need to specify `tagName: 'div'` -- it is the default, and comes for free from Ember.View and Ember.Component